include/constants/union_room.h

```diff
@@ -0,0 +1,28 @@
+#ifndef GUARD_CONSTANTS_UNION_ROOM_H
+#define GUARD_CONSTANTS_UNION_ROOM_H
+
+#define LINK_GROUP_SINGLE_BATTLE        0
+#define LINK_GROUP_DOUBLE_BATTLE        1
+#define LINK_GROUP_MULTI_BATTLE         2
+#define LINK_GROUP_TRADE                3
+#define LINK_GROUP_POKEMON_JUMP         4
+#define LINK_GROUP_BERRY_CRUSH          5
+#define LINK_GROUP_BERRY_PICKING        6
+#define LINK_GROUP_WONDER_CARD          7
+#define LINK_GROUP_WONDER_NEWS          8
+#define LINK_GROUP_UNK_9                9
+#define LINK_GROUP_UNK_10               10
+#define LINK_GROUP_UNK_11               11
+#define LINK_GROUP_RECORD_CORNER        12
+#define LINK_GROUP_BERRY_BLENDER        13
+#define LINK_GROUP_UNK_14               14
+#define LINK_GROUP_COOL_CONTEST         15
+#define LINK_GROUP_BEAUTY_CONTEST       16
+#define LINK_GROUP_CUTE_CONTEST         17
+#define LINK_GROUP_SMART_CONTEST        18
+#define LINK_GROUP_TOUGH_CONTEST        19
+#define LINK_GROUP_BATTLE_TOWER         20
+#define LINK_GROUP_BATTLE_TOWER_OPEN    21
```

Comment:
Are these even used in this project?

Reply:
Based on union room data (e.g. `gUnknown_84591B8` compared to pokeemerald's `sChooseTrainerTexts`), no. Wireless was new for FRLG too so there wouldnt be dummy references from R/S, which I forgot.